cortx-monitorQuickstartGuide.md

```diff
@@ -103,57 +72,119 @@ This guide provides a step-by-step walkthrough for getting you CORTX-Monitor Ser
+        ```
+        
+        If you check the location `/var/artifacts/0/`, you will see that Cortx RPMs and 3rd party packages are generated.
+
+        ```shell
+        ll /var/artifacts/0/
+        The system output displays as follows:
+
+        [root@ssc-vm-2699 ~]# ll /var/artifacts/0/
+        total 1608308
+        drwxr-xr-x  13 root root       4096 Aug 19 05:16 3rd_party
+        drwxr-xr-x   3 root root       4096 Aug 19 05:16 cortx_iso
+        -rw-r--r--   1 root root      19416 Aug 19 05:16 install-2.0.0-0.sh
+        drwxr-xr-x 198 root root       4096 Aug 19 05:16 python_deps
+        -rw-r--r--   1 root root  241635505 Jun 15 08:45 python-deps-1.0.0-0.tar.gz
+        -rw-r--r--   1 root root 1405229297 Jul  1 22:45 third-party-1.0.0-0.tar.gz
+        ```
     
-        `/root/rpmbuild/RPMS/noarch/` and,
-     
-        `/root/rpmbuild/RPMS/x86_64/`
+3. Install CORTX-Monitor RPMs
+
+   SSPL has a `sspl_dev_deploy` script. We need to download and use it to deploy SSPL. Go to the home directory and run following commands.
 
-    **Sample Output:**
-    
     ```shell
+    $ curl https://raw.githubusercontent.com/Seagate/cortx-monitor/main/low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy -o sspl_dev_deploy
+    $ chmod a+x sspl_dev_deploy
+    ```
+    **sspl_dev_deploy is only meant for dev purpose and to install SSPL independently along with its dependencies.**
 
-    [local_host cortx-monitor]# ls -lrt /root/rpmbuild/RPMS/noarch/
-    total 59056
-    -rw-r--r-- 1 root root 34025516 Aug 18 07:26 cortx-sspl-1.0.0-1_git8907300.el7.noarch.rpm
-    -rw-r--r-- 1 root root 16795340 Aug 18 07:27 cortx-sspl-cli-1.0.0-1_git8907300.el7.noarch.rpm
-    -rw-r--r-- 1 root root  9643028 Aug 18 07:27 cortx-sspl-test-1.0.0-1_git8907300.el7.noarch.rpm
-
-
-    [local_host cortx-monitor]# ls -lrt /root/rpmbuild/RPMS/x86_64/
-    total 284
-    -rw-r--r-- 1 root root  65968 Aug 18 07:23 systemd-python36-1.0.0-1_git8907300.el7.x86_64.rpm
-    -rw-r--r-- 1 root root  60904 Aug 18 07:23 systemd-python36-debuginfo-1.0.0-1_git8907300.el7.x86_64.rpm
-    -rw-r--r-- 1 root root   6552 Aug 18 07:27 cortx-libsspl_sec-1.0.0-1_git8907300.el7.x86_64.rpm
-    -rw-r--r-- 1 root root  28448 Aug 18 07:27 cortx-libsspl_sec-debuginfo-1.0.0-1_git8907300.el7.x86_64.rpm
-    -rw-r--r-- 1 root root   5760 Aug 18 07:27 cortx-libsspl_sec-method_none-1.0.0-1_git8907300.el7.x86_64.rpm
-    -rw-r--r-- 1 root root   9592 Aug 18 07:27 cortx-libsspl_sec-method_pki-1.0.0-1_git8907300.el7.x86_64.rpm
-    -rw-r--r-- 1 root root 101004 Aug 18 07:27 cortx-libsspl_sec-devel-1.0.0-1_git8907300.el7.x86_64.rpm
+4. Deploy SSPL RPMs.
+    ```
+    $ ./sspl_dev_deploy --cleanup
+    $ ./sspl_dev_deploy --prereq -T file:///var/artifacts/0
+    ```
+    :page_with_curl: **Note:** 
+    --cleanup includes removing cortx-py-utils RPM. Any other component RPMS installed on the setup will also get removed while removing cortx-py-utils.
```

Comment:
Can we modify sspl_dev_deploy on this? So that no dependency removed as part of cortx-py-utils removal.
`rpm -e --nodeps cortx-py-utils`

Reply:
But still Note is required on --cleanup. Because it cleans yum repos.

Reply:
It is better to erase dependencies also. Because if later you want to deploy specific build number then, it is better to install dependencies from that build itself.

Reply:
Im fine then unless no harm to setup or other components.